data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,293 @@
+	name
+		word
+			"Group of Wanderers"
+	message
+		word
+			"While walking through the spaceport, you see several Wanderers flying through the air, about to land. They do not notice you, and you duck to avoid them."
+			"A crowd of Wanderers has formed around an object. Upon closer inspection, it appears to be some sort of technological device."
+			"As you walk by a group of Wanderers, they begin to coo and chitter at you. Evidently they have not seen something like you before."
+			"A group of Wanderer children appear to be playing a game similar to hopscotch, but instead of hopping they fly to their spots."
+			"On a booth near the spaceport, a group of Wanderers are playing on objects that look vaguely like wooden flutes. Their music is quiet and lacks a discernible melody."
+			"Two Wanderers are at a table. They appear to be playing some sort of board game using shell pieces."
+			"A crowd of Wanderer children follow an adult through the spaceport. The adult talks to them as they follow, as the children take notes."
+			
+news "Wanderers and Korath"
+	to show
+		has "Wanderers: Sestor: Final: done"
+	location
+			government "Wanderer"
+			not attributes "evacuation"
+	name 
+		word
+			"Wanderer merchant captain"
+			"Wanderer captain"
+			"Wanderer"
+			"Wanderer civilian"
+	message
+		word
+			`"`
+		word
+			"The Korath have [enormous, gigantic] ships that many of them live on their whole lives. I cannot [imagine, fathom] doing the same."
+			"Have you [seen, viewed] a Korath up close? They only have feathers on their neck. How [strange, odd]."
+			"The engines the Korath use have no regard for [effciency, subtlety]. But I must admit, they are much more powerful than ours."
+			"The Korath cannot pay [well, adequately] for their transport jobs, but they insist on paying [despite it, nonetheless]."
+			"I am trying to [learn, understand] more words of the Korath language. It is a difficult [task, undertaking], but it is important to learn the language of those who we are [undertaking, rescuing]."
+			"Have you been on a Korath [world, planet]? It is a [sobering, depressing] reminder of the importance of our [task, work]."
+			"I am [excited, overjoyed] to be part of [healing, renewing] the Korath worlds. We are once again [transforming, altering] the old into the new."
+		word
+			`"`
+	name
+		word
+			"Wanderer linguist"
+			"Linguist"
+	message
+		word
+			`"`
+		word
+			"Many scientists are currently [studying, examining] the Korath language, and attempting to teach it to others. It is [exciting, satisfying] to be part of this [effort, undertaking]."
+			"We are still [figuring out, determining] whether or not a Wanderer who has not undergone the Molt could [learn, apprehend] the Korath language. The sounds in their [speech, tongue] are hard for us to [speak, pronounce]."
+		word
+			`"`
+			
+news "Wanderers and Aliens"
+	to show
+		has "Wanderers Conversation: offered"
```

Comment:
It might make more sense to use the following condition. This could be done for the other news entries below this.
```suggestion
		has "language: Wanderers"
```